src/edwards.rs

```diff
@@ -123,7 +123,6 @@ impl CompressedEdwardsY {
     }
 
     /// Copy this `CompressedEdwardsY` to an array of bytes.
-    /// XXX is this useful?
```

Comment:
Did we figure out whether it makes sense to have both `as_bytes` and `to_bytes`?

Reply:
I think it's nice syntactic sugar, and I'm using both in other crates? Also it is nice because it lets users know it's okay to do something, because anytime you're doing a `.0` somewhere you should feel like you're touching something you shouldn't.